src/cli/cli.cpp

```diff
@@ -2091,6 +2091,35 @@ otError Interpreter::ProcessIpAddr(Arg aArgs[])
         {
             OutputIp6AddressLine(*otThreadGetMeshLocalEid(GetInstancePtr()));
         }
+        else if (aArgs[0] == "-v")
+        {
+            const otNetifAddress *unicastAddrs = otIp6GetUnicastAddresses(GetInstancePtr());
+
+            for (const otNetifAddress *addr = unicastAddrs; addr; addr = addr->mNext)
+            {
+                OutputIp6Address(addr->mAddress);
+                OutputFormat(" Origin:");
```

Comment:
```suggestion
                OutputFormat(" origin:");
```
nit: keep everything in lowercase?